content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,158 @@
+
+Ingestion mechanisms are controllable from the Datadog Agent and from Datadog tracing libraries. If you are using OpenTelemetry SDKs to instrument your applications, read [Ingestion Sampling with OpenTelemetry][2].
+
+This guide helps you understand when and how to use ingestion control configurations depending on the main use cases you might encounter. It covers:
+
+- [Determining which ingestion mechanisms are used](#determining-which-ingestion-mechanisms-are-used) for a given service
+- [Use cases that focus on keeping particular types of traces](#keeping-certain-types-of-traces)
+- [Use cases that focus on reducing ingested traces](#reducing-ingestion-for-high-volume-services)
+
+
+## Determining which ingestion mechanisms are used
+
+To identify what ingestion mechanisms are currently used in your Datadog environment, navigate to the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes *by service*. The Configuration column provides a first indication of the current set up. It shows: 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied to the traces that start from the service. Read more about the specifics of [Datadog Agent ingestion logic][5].
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied to the traces that start from the service.
+
+Click on services to see details about what sampling decision makers are used for each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/service_ingestion_summary.png" alt="Service Ingestion Summary" style="width:90%;" >}}
+
+## Keeping certain types of traces
+
+### Keeping entire transaction traces
+
+#### Why ? 
+
+Ingesting entire transaction traces ensures to keep visibility over the **end-to-end service request flow** for specific individual requests.
+
+#### What solutions does Datadog provide ?
+
+Complete traces can be ingested with [head-based sampling][4] mechanisms: the decision of keeping or dropping the trace is taken at the trace creation (trace root span), and this decision is propagated in the request context to downstream callee services.
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/head_based_sampling_keep.png" alt="Head-based Sampling" style="width:100%;" >}}
+
+To decide which traces to keep and drop, the Datadog Agent automatically computes sampling rates for each service to apply at the trace creation, based on the application traffic.
+- For low-traffic applications, a sampling rate of 100% is applied
+- For high-traffic applications, a lower sampling rate is applied
```

Comment:
"a lower sampling rate is applied"
Can we be more specific here? Are there defaults?